qt/place_page_dialog_user.cpp

```diff
@@ -22,6 +22,21 @@ namespace
 static int constexpr kMaxLengthOfPlacePageDescription = 500;
 static int constexpr kMinWidthOfShortDescription = 390;
 
+std::string getShortDescription(std::string description)
```

Comment:
There are many memory allocations here for string processing, but... ok.

Reply:
Addressed using string_view. IMO not as pretty, but it's two less allocations.